Tella/Data/MainAppModel.swift

```diff
@@ -108,6 +132,26 @@ class MainAppModel: ObservableObject, AppModelFileManagerProtocol {
         publishUpdates()
     }
     
+    func deleteAfterMaxAttempts() {
+        let fileManager = FileManager.default
```

Comment:
@ValbuenaG "Could you add a function in Vaultmanager that can delete the content of document folder"

Yes you need to move the logic to vaultmanager because we have already the `containerURL` and we have `fileManager` that contains already `removeItem` function